eos/constraints/B-to-V-form-factors.yaml

```diff
@@ -324,6 +324,208 @@ B->D^(*)::FormFactors[parametric,LCSRLattice]@GKvD:2018A:
     - [ 0.0008983905959576214, 0.007433164049084373, -0.00037584071774735656, 0.007854136708551805, 0.03181878070094583, 0.0003283227396411212, 0.0009025395770889408, -0.0059155556783413565, 0.004709371454878413, 0.013009791658700842, -0.26562583465264444, 0.004588527369236015, 0.04905271440276202, -0.5830194792465329, 0.025385325112283478, -0.3683310413390311, 0.006164768120498282, -0.01567997438885688, 0.021681476451420652, 0.00534579982774948, -0.01070257026098783, 0.0012504283364970527, 0.023686297684736513, -0.07576080744404473, 0.008594328967575927, -0.103365907615185, 1.34541826197464 ]
     - [ 0.0018799166774579076, 0.01330705458253979, 0.0473719660087457, 0.01780101157714347, 0.08584904042758762, -8.253643204185081e-07, 0.042838946498158746, -0.676135196272525, 0.0053999690646070325, 0.05809979677518654, -0.6117215995376126, 0.005623188131591314, 0.07134706358514274, -0.5594588408223317, 0.02187874273366929, -0.15888287487158687, 0.0077943224028272, -0.011671358755047474, 0.07918174700900822, 0.006784608353340076, -0.054718290721080025, 0.450621395927106, 0.059149310917466266, -0.5276480399201693, -0.103365907615185, 13.28158843792259, -199.87023760896412 ]
     - [ 0.029851543058780197, 0.31170747376330454, -0.6857543519222005, 0.2440600245159593, 0.5933214436013504, 0.010448234922404919, -0.3079350760906485, 5.895296552384917, 0.05702769942028968, 0.004649178048391684, -1.1840508344005505, 0.06125472267922745, 0.6232552722719916, -7.548389919026471, 0.37320882924944515, -4.562995703107296, 0.10192759098709293, -0.3193910687938254, -2.0154118175372098, 0.07716850529235574, 0.48814836550821467, -5.505099977968204, -0.15391103660598557, 7.563832150621461, 1.34541826197464, -199.87023760896412, 3297.3143471578883 ]
+  dof: 27
```

Comment:
`dof` is now an optional parameter, it will not be parsed https://github.com/eos/eos/blob/b0842a66124ce920cf9f95ce86d6d81568e6342a/eos/constraint.cc#L1293

Reply:
Done.